nightingale_interface/screens/item_fill_screen.py

```diff
@@ -0,0 +1,134 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+from screens.screen_config import ScreenConfig as cfg
+
+
+class ItemFillScreen:
+    item_fill_name = "itemfillscreen"
+
+    # screen id from the screen manager
+    item_fill_screen_id = 10
+
+    def estop(self, button_data):
```

Comment:
estop function should only be defined once. If the same function is used in multiple screens, it should be stubbed in the screen file and overridden in main. See the call_ros_action function (in main and in confirmation_screen) from my pr for more info

Reply:
Do you have any suggestions on where to implement it once?